frontend/src/pages/Courses.jsx

```
@@ -58,15 +58,15 @@ const Courses = () => {
 
   return (
     <div className={styles.container}>
-      <div className={styles.card} onClick={() => setShowForm(true)}>
+      <button className={styles.card} onClick={() => setShowForm(true)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)